iBudget/fund/views.py

```diff
@@ -21,3 +29,60 @@ def show_fund(request):
             user_funds.append({'id': entry.id, 'name': entry.name})
         return JsonResponse(user_funds, status=200, safe=False)
     return JsonResponse({}, status=400)
+
+
+@require_http_methods(["GET"])
+def show_fund_group(request):
```

Comment:
You have called method `show_fund_group` but it returns a list of `users_fund`. This is confusing.